Fonts/PTSenif-Bold/specimen_files/specimen_stylesheet.css

```diff
@@ -0,0 +1,463 @@
+  font: 11px/12px 'Courier New', Courier;
+}
+
+code {
+  white-space: pre;
+  background-color: #eee;
+  display: block;
+  padding: 10px;
+  margin-bottom: 18px;
+  overflow: auto;
+}
+
+.bottom,
+.last {
+  margin-bottom: 0;
+  padding-bottom: 0;
+}
+
+.box {
+  padding: 18px;
+  margin-bottom: 18px;
+  background: black;
+}
+
+.reverse,
+.reversed {
+  background: black;
+  color: white;
+  border: none;
+}
+
+#bodycomparison {
+  position: relative;
+  overflow: hidden;
+  font-size: 72px;
+  height: 90px;
+  white-space: nowrap;
+}
+
+#bodycomparison div {
+  font-size: 72px;
+  line-height: 90px;
+  display: inline;
+  margin: 0 15px 0 0;
+  padding: 0;
+}
+
+#bodycomparison div span {
+  font: 10px;
+  position: absolute;
+  left: 0;
+}
+
+#xheight {
+  float: none;
+  position: absolute;
+  color: #d9f3ff;
+  font-size: 72px;
+  line-height: 90px;
+}
+
+.fontbody {
+  position: relative;
+}
+
+.arialbody {
+  font-family: Arial;
+  position: relative;
+}
+
+.verdanabody {
+  font-family: Verdana;
+  position: relative;
+}
+
+.georgiabody {
+  font-family: Georgia;
+  position: relative;
+}
+
+/* @group Layout page
+ */
+
+#layout h1 {
+  font-size: 36px;
+  line-height: 42px;
+  font-weight: normal;
+  font-style: normal;
+}
+
+#layout h2 {
+  font-size: 24px;
+  line-height: 23px;
+  font-weight: normal;
+  font-style: normal;
+}
+
+#layout h3 {
+  font-size: 22px;
+  line-height: 1.4em;
+  margin-top: 1em;
+  font-weight: normal;
+  font-style: normal;
+}
+
+#layout .byline p {
+  font-size: 12px;
+  margin-top: 18px;
+  line-height: 12px;
+  margin-bottom: 0;
+}
+
+#layout p { font-size: 14px; line-height: 21px; margin-bottom: .5em; }
+
+#layout .large p {
+  font-size: 18px;
+  line-height: 26px;
+}
+
+#layout .sidebar p {
+  font-size: 12px;
+  line-height: 1.4em;
+}
+
+#layout .caption p {
+  font-size: 10px;
+  margin-top: -16px;
+  margin-bottom: 18px;
+}
+
+/* @end */
+
+/* @group Glyphs */
+
+#glyph_chart div {
+  background-color: #d9f3ff;
+  color: black;
+  float: left;
+  font-size: 36px;
+  height: 1.2em;
+  line-height: 1.2em;
+  margin-bottom: 1px;
+  margin-right: 1px;
+  text-align: center;
+  width: 1.2em;
+  position: relative;
+  padding: .6em .2em .2em;
+}
+
+#glyph_chart div p { position: absolute;
+  left: 0;
+  top: 0;
+  display: block;
+  text-align: center;
+  font: bold 9px Arial, sans-serif;
+  background-color: #3a768f;
+  width: 100%;
+  color: white;
+  padding: 2px 0;
+}
+
+#glyphs h1 {
+  font-family: Arial, sans-serif;
+}
+
+/* @group Installing */
+
+#installing {
+  font: 13px Arial, sans-serif;
+}
+
+#installing p { line-height: 1.2em; margin-bottom: 18px; font: 13px Arial, sans-serif; }
+
+#glyphs p { line-height: 1.2em; margin-bottom: 18px; font: 13px Arial, sans-serif; }
+
+#installing h3 {
+  font-size: 15px;
+  margin-top: 18px;
+}
+
+/* @end */
+
+#rendering h1 {
+  font-family: Arial, sans-serif;
+}
+
+.render_table td { font: 11px "Courier New", Courier, mono; vertical-align: middle; }
```

Comment:
Codacy found an issue: [Expected selector ".render_table td" to come before selector ".sample_table td:first-child" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/-Kostikova-Irina/pullRequest?prid=6241956)